.circleci/config.yml

```diff
@@ -475,50 +475,50 @@ workflows:
               only: master
           requires:
               - npm_install
-  tap_tests:
-    triggers:
-      - schedule:
-          # Everyday @ 6:15PM UTC | 3:15PM EST | 2:15PM EDT | 12:15PM PST | 11:15AM PDT
-          cron: '15 18 * * 1-5'
-          filters:
-            branches:
-              only:
-                - master
-    jobs:
-      - npm_install:
-          filters:
-              branches:
-                only: master
-      - build_production:
-          requires:
-            - npm_install
-      - tap:
-          name: '@webex/webex-core tap'
-          package: '@webex/webex-core'
-          requires:
-            - build_production
-      - tap:
-          name: '@webex/plugin-meetings tap'
-          package: '@webex/plugin-meetings'
-          requires:
-            - build_production
-      - tap:
-          name: '@webex/internal-plugin-conversation tap'
-          package: '@webex/internal-plugin-conversation'
-          requires:
-            - build_production
-      - tap:
-          name: '@webex/internal-plugin-encryption tap'
-          package: '@webex/internal-plugin-encryption'
-          requires:
-            - build_production
-      - tap:
-          name: '@webex/internal-plugin-locus tap'
-          package: '@webex/internal-plugin-locus'
-          requires:
-            - build_production
-      - tap:
-          name: '@webex/internal-plugin-mercury tap'
-          package: '@webex/internal-plugin-mercury'
-          requires:
-            - build_production
+  # tap_tests:
+  #   triggers:
+  #     - schedule:
+  #         # Everyday @ 6:15PM UTC | 3:15PM EST | 2:15PM EDT | 12:15PM PST | 11:15AM PDT
+  #         cron: '15 18 * * 1-5'
+  #         filters:
+  #           branches:
+  #             only:
+  #               - master
```

Comment:
I'd just comment/remove the trigger section

Reply:
Done